eventmesh-runtime/src/test/java/org/apache/eventmesh/runtime/admin/handler/RedirectClientByPathHandlerTest.java

```diff
@@ -124,7 +124,7 @@ public void testHandle() throws IOException {
                 clientMockedStatic.when(() -> EventMeshTcp2Client.redirectClient2NewEventMesh(any(), anyString(), anyInt(), any(),
                     any())).thenThrow(new RuntimeException());
                 redirectClientByPathHandler.handle(mockExchange);
-                String response = outputStream.toString();
+                String response = outputStream.toString(StandardCharsets.UTF_8.name());
```

Comment:
replace with Constants.DEFAULT_CHARSET

Reply:
Hi @mxsm, In that case should I make the changes on Line 107 and 117 of the same file where the format is similar to the one I submitted for PR.

Reply:
![image](https://user-images.githubusercontent.com/124816912/230725433-d1cff15f-eee9-40f8-a7ae-de08a7f43f1d.png)
![image](https://user-images.githubusercontent.com/124816912/230725478-382e249a-d587-4d62-b617-f9566e1820d3.png)

Reply:
Hi @mxsm / @Alonexc / @xwm1992 - Should I go ahead with the above mentioned changes as well?

I.e. replacing StandardCharsets.UTF_8 with Constants.DEFAULT_CHARSET on line 107 and 117 along with the changes made for this issue on line 128 [RedirectClientByPathHandlerTest]